include/config/overworld.h

```diff
@@ -42,9 +42,10 @@
                                                   // Compressed gfx are incompatible with non-power-of-two sprite sizes:
                                                   // (You should not use 48x48 sprites/tables for compressed gfx)
                                                   // 16x32, 32x32, 64x64 etc are fine
+#define OW_MON_BOBBING                 TRUE       // If true, OW pokemon will bob up and down during their idle & walking animations
+#define OW_MON_WANDER_WALK             TRUE       // If true, OW pokemon with MOVEMENT_TYPE_WANDER will walk-in-place in between steps.
```

Comment:
can both of those be true at the same time?

Reply:
Yes, I believe if they are both true at the same time then it mimics what following pokemon do